src/API/Site/Controllers/RestAPI/AuthController.php

```diff
@@ -105,6 +156,23 @@ protected function get_auth_params(): array {
 		];
 	}
 
+	/**
+	 * Get the query params for the update authorize request.
+	 *
+	 * @return array
+	 */
+	protected function get_update_authorize_params(): array {
+		return [
+			'status' => [
+				'description'       => __( 'The status of the merchant granting access to Google\'s WPCOM app', 'google-listings-and-ads' ),
+				'type'              => 'string',
+				'enum'              => [ 'approved', 'disapproved', 'error' ],
```

Comment:
What do you think about making these enums constants of the `OAuthService class`? That way, if we need to check the status later, we could simply use something like `OAuthService::APPROVED` instead of typing out the values each time.

Reply:
Thanks for the suggestion, updated in f19b5f7.